### PR TITLE
Add ParadeDB search benchmark blog post

### PR DESCRIPTION
**Pull request by jasonmp85**
Need an image to go with this post, but otherwise the content and benchmarks are good for review.

### CODE REVIEW

File: src/content/blog/2024-11-07-parade-search/index.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## ParadeDB's `pg_search`
+
+But as we've seen time and time again, PostgreSQL is flexible enough to absorb these use cases whole: why shuffle data off to some external indexer instead of searching it in the database already storing it?
+
+A particular weakness of PostgreSQL's search implementation is that the ranking functions are not generally indexable. Because of this, changing a query from "show me five results" to "show me the top five relevant results" can have poor scaling characteristics. This is the use case we'll highlight in this post.
+
+## Benchmarking
+
+For this benchmark, I used ParadeDB's own [`cargo-paradedb`](https://github.com/paradedb/paradedb/tree/dev/cargo-paradedb) tool to generate workloads and queries. It wraps a Go tool which generates the Elastic OpenSearch Benchmark corpus and which can create GIN and BM25 indexes for benchmarking.
```
Comment (FloorD) [inline src/content/blog/2024-11-07-parade-search/index.mdx:28]: Elasticsearch vs OpenSearch benchmark?

File: src/content/blog/2024-11-07-parade-search/index.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+The default search query simply requests a certain number of matches. While useful, I believe sorting by some sort of rank is more representative of real-world use. To that end, I [modified](https://github.com/jasonmp85/paradedb/commit/f6c7ca1b39b1e92ee4c2717368d5df352100e050) the tool's SQL to include a "top N"-like sort.
```
Comment (FloorD) [inline src/content/blog/2024-11-07-parade-search/index.mdx:30]: Maybe bring this over to Tembo Labs instead?

File: src/content/blog/2024-11-07-parade-search/index.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+#### Full Text Search
+
+A query to find the top ten documents matching a search string took an average (mean) of **1.66 seconds** to complete.
+
+![Full Text Search Results, 10MM Rows](./GIN-10MM-PDF.svg)
+
+#### ParadeDB's `pg_search`
+
+The ParadeDB implementation answered a comparable query within **6.28ms**, _nearly 265 times faster_. Note that the probability distribution function reflects that these results are much more consistent than the full text ones, as well…
```
Comment (FloorD) [inline src/content/blog/2024-11-07-parade-search/index.mdx:46]: "as well…" the way that sentence is structured, it feels unfinished?

File: src/content/blog/2024-11-07-parade-search/index.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+#### ParadeDB's `pg_search`
+
+`pg_search` retains a commendable lead, increasing to around **28ms**, nearly **500x** the performance of PostgreSQL's full text search.
+
+![pg_search Results, 10MM Rows](./search-100MM-PDF.svg)
+
+## Conclusion
+
+Though PostgreSQL includes a full-featured text search capability, the BM25 index type used by ParadeDB's `pg_search` vastly improves performance in applications which need to list search results by rank or relevance. Tembo's ParadeDB stack provides an easy way for users to deploy and use this advanced search functionality in their own applications today.
```
Comment (FloorD) [inline src/content/blog/2024-11-07-parade-search/index.mdx:68]: Maybe include a link to our ParadeDB launch blog post, or docs? https://tembo.io/blog/paradedb-search-analytics https://tembo.io/docs/product/stacks/analytical/paradedb

File: src/content/blog/2024-11-07-parade-search/index.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+slug: paradedb-search
+title: 'pg_search from ParadeDB: Benchmarks'
+authors: [jason]
+description: |
+    Speed up your full text search with pg_search from ParadeDB!
+tags: [tembo, search, paradedb, benchmarking]
```
Comment (FloorD) [inline src/content/blog/2024-11-07-parade-search/index.mdx:7]: elasticsearch, opensearch, postgres ?

File: src/content/blog/2024-11-07-parade-search/index.mdx [hunk 6]
```diff
@@ -0,0 +1,68 @@
+---
+slug: paradedb-search
+title: 'pg_search from ParadeDB: Benchmarks'
```
Comment (FloorD) [inline src/content/blog/2024-11-07-parade-search/index.mdx:3]: Benchmarking ParadeDB's pg_search: Postgres as your search engine

**Review by FloorD — APPROVED** [review]

File: src/content/blog/2024-11-07-parade-search/index.mdx (hunk 1 above)
Comment (jasonmp85) [reply src/content/blog/2024-11-07-parade-search/index.mdx:28]: Oops, I wrote that from the URL slug and not the actual expanded title. Fixing.

File: src/content/blog/2024-11-07-parade-search/index.mdx (hunk 4 above)
Comment (jasonmp85) [reply src/content/blog/2024-11-07-parade-search/index.mdx:68]: That's a good idea. I'll link _ParadeDB Stack_ to the docs.

File: src/content/blog/2024-11-07-parade-search/index.mdx (hunk 5 above)
Comment (jasonmp85) [reply src/content/blog/2024-11-07-parade-search/index.mdx:7]: 👍🏻

File: src/content/blog/2024-11-07-parade-search/index.mdx (hunk 6 above)
Comment (jasonmp85) [reply src/content/blog/2024-11-07-parade-search/index.mdx:3]: Thanks :-P

**Pull request merged**